apps/web/store/bulkBookmarksAction.tsx

```diff
@@ -0,0 +1,44 @@
+// reference article https://refine.dev/blog/zustand-react-state/#build-a-to-do-app-using-zustand
+import { create } from "zustand";
+
+import type { ZBookmark } from "@hoarder/shared/types/bookmarks";
+
+interface BookmarkState {
+  selectedBookmarks: ZBookmark[];
+  isBulkEditEnabled: boolean;
+  handleBulkEdit: (isEnabled: boolean) => void;
+  toggleBookmark: (bookmark: ZBookmark) => void;
+  getSelectedBookmarks: () => ZBookmark[];
```

Comment:
What's the point of this getter? Why can't we just use `selectedBookmarks` directly?